GMessenger/Messenger.Database/Configurations/FriendConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Messenger.Database.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Messenger.Database.Configurations
+{
+    public class FriendConfiguration : IEntityTypeConfiguration<Friend>
+    {
+        public void Configure(EntityTypeBuilder<Friend> builder)
+        {
+            builder
+            .HasOne(b => b.Users)
+            .WithMany()
+            .HasForeignKey(b => b.UserId);
+
+            builder
+            .HasOne(b => b.UserFriends)
+            .WithMany()
```

Comment:
Reikia pataisyti? https://www.entityframeworktutorial.net/efcore/configure-one-to-many-relationship-using-fluent-api-in-ef-core.aspx

Reply:
yes, siulyciau pakoreguoti